lib/usdUfe/ufe/UsdUndoAddNewPrimCommand.h

```diff
@@ -67,6 +68,10 @@ class USDUFE_PUBLIC UsdUndoAddNewPrimCommand : public Ufe::UndoableCommand
 
 }; // UsdUndoAddNewPrimCommand
 
+//! \brief Retrieve the desired selection after the command has executed.
+//         \see UsdUndoSelectAfterCommand.
+Ufe::Selection USDUFE_PUBLIC getNewSelectionFromCommand(const UsdUndoAddNewPrimCommand& cmd);

```

Comment:
Why is this not inside the class? Seems like a very generic function name to be in the UsdUfe namespace directly.

Reply:
Because it is a function that needs to be overloaded for each specific command class to be callable by the template. It also allows (Like for the InsertChildCommand) to be added to a class to which we don't have the source code in MayaUsd. I choose a long function name in order to avoid possible name conflicts.